vertx-grpc-common/src/main/java/io/vertx/grpc/common/impl/BridgeMessageDecoder.java

```diff
@@ -29,15 +33,31 @@ public BridgeMessageDecoder(MethodDescriptor.Marshaller<T> marshaller, Decompres
     this.decompressor = decompressor;
   }
 
+  private static class KnownLengthStream extends ByteBufInputStream implements KnownLength {
+    public KnownLengthStream(Buffer buffer) {
+      super(((BufferImpl)buffer).getByteBuf(), buffer.length());
```

Comment:
the buffer is doesn't have any weird requirement re release ref cnt?

Reply:
it does create a slice but closing the ByteBufInputStream releases it